balancer/rls/cache_test.go

```diff
@@ -242,3 +242,50 @@ func (s) TestDataCache_ResetBackoffState(t *testing.T) {
 		t.Fatalf("unexpected diff in backoffState for cache entry after dataCache.resetBackoffState(): %s", diff)
 	}
 }
+
+var cacheEntriesMetricsTests = []*cacheEntry{
```

Comment:
Please move inside test function so it's not global.

Reply:
Ah good point. Done.